src/reducer.js

```diff
@@ -206,54 +206,80 @@ function reducer(
-      obligatory_user_fields: null,
-      errorL1s: null,
-    };
-  case "OBLIGTORY_USER_FIELDS_RESP":
-    console.log("USER FILED RESPONSE ", action.payload.data.userObligatoryFields)
-    return {
-      ...state,
-      fetching_obligatory_user_fields: false,
-      fetched_obligatory_user_fields: true,
-      obligatory_user_fields: action.payload.data.userObligatoryFields,
-      errorL1s: formatGraphQLError(action.payload),
-    };
-  case "OBLIGTORY_USER_FIELDS_ERR":
-    return {
-      ...state,
-      fetching_obligatory_user_fields: false,
-      errorL1s: formatServerError(action.payload),
-    };
-  case "OBLIGTORY_EO_FIELDS_REQ":
-    return {
-      ...state,
-      fetching_obligatory_eo_fields: true,
-      fetched_obligatory_eo_fields: false,
-      obligatory_eo_fields: null,
-      errorL1s: null,
-    };
-  case "OBLIGTORY_EO_FIELDS_RESP":
-    console.log("EO FILED RESPONSE ", action.payload.data.eoObligatoryFields);
-    return {
-      ...state,
-      fetching_obligatory_eo_fields: false,
-      fetched_obligatory_eo_fields: true,
-      obligatory_eo_fields: action.payload.data.eoObligatoryFields,
-      errorL1s: formatGraphQLError(action.payload),
-    };
-  case "OBLIGTORY_EO_FIELDS_ERR":
-    return {
-      ...state,
-      fetching_obligatory_eo_fields: false,
-      errorL1s: formatServerError(action.payload),
-    };
+      return {
+        ...state,
+        fetching_obligatory_user_fields: true,
+        fetched_obligatory_user_fields: false,
+        obligatory_user_fields: null,
+        errorL1s: null,
+      };
+    case "OBLIGTORY_USER_FIELDS_RESP":
+      console.log("USER FILED RESPONSE ", action.payload.data.userObligatoryFields);
+      return {
+        ...state,
+        fetching_obligatory_user_fields: false,
+        fetched_obligatory_user_fields: true,
+        obligatory_user_fields: action.payload.data.userObligatoryFields,
+        errorL1s: formatGraphQLError(action.payload),
+      };
+    case "OBLIGTORY_USER_FIELDS_ERR":
+      return {
+        ...state,
+        fetching_obligatory_user_fields: false,
+        errorL1s: formatServerError(action.payload),
+      };
+    case "OBLIGTORY_EO_FIELDS_REQ":
+      return {
+        ...state,
+        fetching_obligatory_eo_fields: true,
+        fetched_obligatory_eo_fields: false,
+        obligatory_eo_fields: null,
+        errorL1s: null,
+      };
+    case "OBLIGTORY_EO_FIELDS_RESP":
+      console.log("EO FILED RESPONSE ", action.payload.data.eoObligatoryFields);
+      return {
+        ...state,
+        fetching_obligatory_eo_fields: false,
+        fetched_obligatory_eo_fields: true,
+        obligatory_eo_fields: action.payload.data.eoObligatoryFields,
+        errorL1s: formatGraphQLError(action.payload),
+      };
+    case "OBLIGTORY_EO_FIELDS_ERR":
+      return {
+        ...state,
+        fetching_obligatory_eo_fields: false,
+        errorL1s: formatServerError(action.payload),
+      };
```

Comment:
Generally I wonder if we should use here in states names 'Pascal case' instead of 'Snake case'

Reply:
This is only a proposal to adjust the convention of naming

Reply:
To do not disturb other functionalities, I changed the convention of naming just in new code.